File: android/app/src/main/java/tut/flightbookingsystem/model/Food.java
package tut.flightbookingsystem.model;

import java.io.Serializable;

public class Food implements Serializable {
    public Long id;
    public String name;
    public String image;
    public String description;
    public long food_type_id;
    public FoodType food_type;
    public String created_at;
    public String updated_at;

    public Food(Long id,
                final String name,
                final String image) {
        this.id = id;
        this.name = name;
        this.image = image;
    }

    @Override
    public String toString() {
        return name;
    }
}

File: web/app/Food.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class Food extends Model
{
  protected $fillable = ['name', 'description', 'food_type_id', 'image'];
  public function food_type(){
    return $this->hasOne('App\FoodType', 'id', 'food_type_id');
  }

  public function getShowLinkAttribute(){
    return route("backoffice.foods.show", $this->id);
  }

  public function getEditLinkAttribute(){
    return route("backoffice.foods.edit", $this->id);
  }

  public function getImageAttribute($image){
    return asset($image);
  }
}
